features/verifysession/impl/src/main/kotlin/io/element/android/features/verifysession/impl/VerifySelfSessionPresenter.kt

```diff
@@ -49,6 +54,7 @@ class VerifySelfSessionPresenter @Inject constructor(
     private val stateMachine: VerifySelfSessionStateMachine,
     private val buildMeta: BuildMeta,
     private val sessionPreferencesStore: SessionPreferencesStore,
+    private val logoutUseCase: LogoutUseCase,
```

Comment:
We should choose at some point bewteen logout or signout, we have a mixed of them currently 😅